firebase-firestore/src/main/java/com/google/firebase/firestore/local/MemoryMutationQueue.java

```diff
@@ -422,4 +422,12 @@ private int indexOfExistingBatchId(int batchId, String action) {
     hardAssert(index >= 0 && index < queue.size(), "Batches must exist to be %s", action);
     return index;
   }
+
+  long getByteSize(LocalSerializer serializer) {
+    long count = 0;
```

Comment:
Can we cache the size of a mutation batch, or maybe even compute them when they are first added (at which point we also need to serialize them to send them to RemoteStore)?